Silicon/Qualcomm/sm8250/Library/PlatformMemoryMapLib/PlatformMemoryMapLib.c

```diff
@@ -0,0 +1,62 @@
+#include <Library/BaseLib.h>
+#include <Library/PlatformMemoryMapLib.h>
+
+static ARM_MEMORY_REGION_DESCRIPTOR_EX gDeviceMemoryDescriptorEx[] = {
+	{"Hypervisor",       0x80000000, 0x000600000, AddMem, SYS_MEM, SYS_MEM_CAP, Reserv, NS_DEVICE},
+	{"HLOS",             0x80600000, 0x000100000, AddMem, SYS_MEM, SYS_MEM_CAP, RtCode, WRITE_BACK_XN},
```

Comment:
Please avoid duplicate names (except for "RAM Partition"). So maybe use "HLOS 1", "HLOS 2" etc